doc/changelog/05-Ltac-language/18094-stratfix.rst

```diff
@@ -0,0 +1,5 @@
+- **Added:**
+  ``rewrite_strat`` strategies now support a fixpoint operator ``fix foo := (bar; foo)``
+  (`#18094 <https://github.com/coq/coq/pull/18094>`_,
```

Comment:
```suggestion
  In :tac:`rewrite_strat`, :n:`@rewstrategy` now supports the fixpoint operator :n:`fix @ident := @rewstrategy1`
  (`#18094 <https://github.com/coq/coq/pull/18094>`_,
```

Two tactics `(bar; foo)` is not the simplest example, nor are `bar` and `foo` valid.

Reply:
> Two tactics `(bar; foo)` is not the simplest example, nor are `bar` and `foo` valid.

`foo` is actually valid in that context, as it's bound by `fix`.  But indeed `bar` is invalid.

Reply:
I guess `tac` should be `tacn`

Reply:
Yes